aten/src/ATen/native/quantized/cpu/qnnpack/test/convolution.cc

```diff
@@ -12,102 +12,93 @@
 
 #include "convolution-operator-tester.h"
 
-TEST(CONVOLUTION_OP, zero_batch) {
+_STATIC_AND_RUNTIME_TEST(CONVOLUTION_OP, zero_batch,
```

Comment:
So we're enabling both static and runtime mode for all conv and deconv tests. Do they all pass? I recall you mentioning some of them failed

Reply:
The same that fail in the regular convolution fail in the transpose as well. I think this is not specific to convolution. Currently, I am enabling the same tests that are enabled from the conv

Reply:
Are you sure? Check this on master? On master I think all convolution tests are passing. If it is not failing on master it should not be failing here.